components/hab/src/cli.rs

```diff
@@ -577,6 +573,24 @@ fn sub_pkg_build() -> App<'static, 'static> {
     }
 }
 
+fn sub_pkg_binlink() -> App<'static, 'static> {
+    let sub = clap_app!(@subcommand binlink =>
+        (about: "Creates a symlink for a package binary in a common 'PATH' location")
+        (@arg PKG_IDENT: +required +takes_value
+            "A package identifier (ex: core/redis, core/busybox-static/1.42.2)")
+        (@arg BINARY: +takes_value
+            "The command to symlink (ex: bash)")
+        (@arg DEST_DIR: -d --dest +takes_value
+            "Sets the destination directory (default: /bin)")
+    );
+    sub.arg(
+        Arg::with_name("FORCE")
+            .help("Overwrites existing files")
+            .short("f")
```

Comment:
A small suggestion: would we be able to use `-F` rather than the lowercase form? I see little `f` and think of “file”.